base_custom_attributes/README.rst

```diff
@@ -0,0 +1,26 @@
+A module add custom attribute for models 
+===========================================
+
+
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Qing Wang <wang.qing@elico-corp.com>
```

Comment:
Isn't the original author of this module `Benoît GUILLOT <benoit.guillot@akretion.com>` ?
Would be fair to add him to this list